src/Plugin/Validation/Constraint/ValidOgMembershipMultipleReferenceConstraint.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\og\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Entity Reference valid references constraint.
+ *
+ * Make sure that when a user that can only post content inside a group
+ * populated one of the audience fields. After the fields are populated the
+ * constraint for checking valid references will make sure the values are OK.
+ *
+ * @Constraint(
+ *   id = "ValidOgMembershipMultipleReference",
+ *   label = @Translation("Organic Groups valid references", context = "Validation")
+ * )
+ */
+class ValidOgMembershipMultipleReferenceConstraint extends Constraint {
+
+  /**
+   * Fields are not populated.
+   *
+   * @var string
+   */
+  public $AudienceFieldsAreNotPopulated = 'One of the fields @fields is required.';
```

Comment:
--------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------
 26 | ERROR | Class property $AudienceFieldsAreNotPopulated should use
    |       | lowerCamel naming without underscores
--------------------------------------------------------------------------------

```suggestion
  public $audienceFieldsAreNotPopulated = 'One of the fields @fields is required.';
```